### ISSUE TITLE
How do I open .cbdb files?

### ISSUE THREAD

**Issue by ghost** [ClassiCube/MCGalaxy#524]
Hello, I know blockdb uses .cbdb extension. I want to see a full list of a map block database, not just clicking hand by hand (would take 1000 years) on a single block.
That's why I need to open it. Is there a way or not?

Thanks, PerLazy

**Comment by Goodlyay** [comment]
What do you even mean by seeing a full list? How would this 3D database of players who placed and deleted blocktypes be represented?

If you want to, you can open .cbdb with a hex editor program, but you still won't see anything human readable from that. 

Might I suggest that what you really want is the command called /highlight? It allows you to pick a username and timespan and highlight all the blocks they deleted and placed during that time.

**Comment by ghost** [comment]
Thanks! But is there a full player one? I mean all players in one

**Comment by Goodlyay** [comment]
No, there isn't. What purpose do you need this for?

**Comment by ghost** [comment]
Or is there a heatmap? (light when new block, gets dark by time the block gets old)

**Comment by UnknownShadow200** [comment]
No, /hightlight just gives you green for blocks a user placed, red for
deleted.

**Comment by rdebath** [comment]
The data is not generally useful outside the in-game undo's (etc) but I did throw together a program that dumps it out into a sort of readable (grep-able) format.

https://github.com/rdebath/mc/blob/master/extras/blockdb.c

I tried slicing it in a few different ways; nothing was particularly interesting, but `-A` gets you every detail.
